java/se/sics/mspsim/cli/WindowTarget.java

```diff
@@ -56,6 +59,7 @@ private void processLine(CommandContext context, String line) {
           dataHandler.setProperty("title", new String[] {args});
         }
       } else if ("type".equals(cmd)) {
+        var hasOldDataHandler = this.dataHandler != null;
```

Comment:
Variable is not used for 10 lines, can it be pushed down closer to the use-site?

Reply:
Not really, as it stores the status of the current data handler that is overwritten in the next statements. However, I removed the `this` prefix, which might be confusing. I personally prefer to prefix global variables with `this` for clarity, but this is usually not done in the Cooja codebase, and I missed this. Thanks for noticing!